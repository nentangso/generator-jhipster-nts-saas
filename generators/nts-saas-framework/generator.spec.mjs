import expect from 'expect';

import { helpers, lookups } from '#test-utils';

const SUB_GENERATOR = 'nts-saas-framework';
const SUB_GENERATOR_NAMESPACE = `jhipster-nts-saas:${SUB_GENERATOR}`;

describe('SubGenerator nts-saas-framework of nts-saas JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create(SUB_GENERATOR_NAMESPACE)
        .withOptions({
          reproducible: true,
          defaults: true,
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
