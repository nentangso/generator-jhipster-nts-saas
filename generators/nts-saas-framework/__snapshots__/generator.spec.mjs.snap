// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator nts-saas-framework of nts-saas JHipster blueprint run should succeed 1`] = `
Object {
  "template-file-base": Object {
    "stateCleared": "modified",
  },
}
`;
